### DISCUSSION TITLE
Как зарегистрировать в survey.ukraine.gov.bg ребёнка без родителей

### DISCUSSION THREAD

**Discussion by Viktoriy2004** [Budibase/budibase#6097] [Help]
Как в https://survey.ukraine.gov.bg/builder/auth/login зарегистрировать ребёнка 13 лет, который находиться в Болгарии без сопровождения родителей. К бабушке в учётную запись не подтягивает внуков. Отдельно создать учётную запись система не даёт на ребёнка. Как зарегистрировать?

**Comment by shogunpurple** [comment]
Здравствуйте @Viktoriy2004 

Извините, я не говорю по-русски, поэтому пользуюсь переводчиком, надеюсь, это понятно.

Мы являемся только поставщиком программного обеспечения, используемого для формы, поэтому я не уверен, как правительство регистрирует пользователей.

Я связался с нашим контактным лицом, и мне сказали, что завтра состоится заседание правительства, на котором будет обсуждаться именно этот вопрос (регистрация несовершеннолетних в приложении).

Вы можете позвонить на национальную горячую линию по телефону +35929055555 для получения дополнительной информации, но, очевидно, завтра правительство даст больше разъяснений по этой ситуации. 

Спасибо
